docker/scripts/java-11-vars.sh

```diff
@@ -2,5 +2,9 @@
 
 JAVA_INSTALL_VERSION_MAJOR='11'
 JAVA_INSTALL_VERSION=3
+MAGIC_B_NUM=12
+JAVA_INSTALL_LINK_HASH=37f5e150db5247ab9333b11c1dddcd30
 JAVA_INSTALL_PATH=/usr/opt/java-${JAVA_INSTALL_VERSION_MAJOR}-${JAVA_INSTALL_VERSION}/
-JDK_FILE=/tmp/jdk-11.0.3_linux-x64_bin.tar.gz
+JDK_VERSION=${JAVA_INSTALL_VERSION_MAJOR}.0.${JAVA_INSTALL_VERSION}
+DOWNLOAD_URL=http://download.oracle.com/otn-pub/java/jdk/${JDK_VERSION}+${MAGIC_B_NUM}/${JAVA_INSTALL_LINK_HASH}/jdk-${JDK_VERSION}_linux-x64_bin.tar.gz
```

Comment:
we should avoid downloading from oracle if possible, every now and then they make a new release and than this link stops working

Reply:
let me reverse this too...